package.json

```diff
@@ -43,6 +43,7 @@
     "eslint-plugin-standard": "^5.0.0",
     "nx": "17.0.3",
     "prettier": "^3.0.3",
+    "semantic-release": "^23.0.0",
```

Comment:
We can remove `semantic-release` all together actually!

Reply:
oh great! I'll do that then :D